src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -424,7 +445,10 @@ def send_error_message(
             other fields about the error, dependent on the content.
         }
         """
-        error_content = {"job_id": req.job_id, "source": req.request}
+        error_content = {
+            job_id_key: getattr(req, job_id_key), 
+            "source": req.request
+        }
```

Comment:
Allow `send_error_message` to send error message with `job_id_list` instead of `job_id` for cases of `job_comm_error`